vnext/Microsoft.ReactNative/Views/KeyboardEventHandler.cpp

```diff
@@ -185,20 +185,36 @@ void HandledKeyboardEventHandler::KeyboardEventHandledHandler(
     args.Handled(true);
 }
 
+bool EqualsKeyboardEvent(HandledKeyboardEvent declaredEvent, HandledKeyboardEvent currentEvent) {
+  return declaredEvent.code == currentEvent.code &&
+      (declaredEvent.ignoreModifiers ||
```

Comment:
Remove ignoreModifiers - was testing out the potential to enable correct default behaviors for single-line text input. If we decide to merge this without single-line TextInput support for submitKeyEvents, we should open an issue to consider it.